resolwe_bio/tools/check_bam_source.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
+"""Check if the source of RNA ae rRNA or globin RNA."""
+
+import argparse
+import os
+import sys
+
+MAPPINGS_DIR = '/opt/chrom_mappings/globin_rrna/'
+QC_FILES = {
+    'Homo sapiens': [
+        'Homo_sapiens_rRNA.txt',
+        'Homo_sapiens_globin_RNA.txt',
+    ],
+    'Mus musculus': [
+        'Mus_musculus_rRNA.txt',
+        'Mus_musculus_globin_RNA.txt'
+    ],
+    'Rattus norvegicus': [
+        'Rattus_norvegicus_rRNA.txt',
+        'Rattus_norvegicus_globin_RNA.txt'
+    ],
+}
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('--chromosome_names', help="File with chromosome names")
+    parser.add_argument('--species', help="Species")
+    return parser.parse_args()
+
+
+def parse_file(filename):
+    """Parse qc files."""
+    qc_file = set()
```

Comment:
This is neither a file, nor it contains a qc sequences. These are chromosome names. Name it `chromosomes` or something `chromosome_names`